app/lib/parachain_manager.py

```diff
@@ -168,19 +170,36 @@ def initialize_parachain(substrate_client, sudo_seed, para_id, state, wasm, leas
                 'period_count': lease_period_count
             }
         ))
+    if force_queue_action:
+        batch_call.append(substrate_client.compose_call(
+            call_module='Paras',
+            call_function='force_queue_action',
+            call_params={
+                'para': para_id
+            }
+        ))
     return substrate_batchall_call(substrate_client, keypair, batch_call, True, True)
 
 
-def cleanup_parachain(substrate_client, sudo_seed, para_id):
+def cleanup_parachain(substrate_client, sudo_seed, para_id, force_queue_action=True):
```

Comment:
force_queue_action already defaults to True in the router. It might be better not to set a default value.

Reply:
I removed default form `offboard_parachain_by_id` function